src/host_vm_loader.rs

```diff
@@ -5,7 +5,7 @@
 use arrayvec::ArrayString;
 use core::{fmt, slice};
```

Comment:
Nit: Typo in commit message (hierarchy).

Reply:
Done.